_data/chains/eip155-5001.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Mantle Testnet",
+  "chain": "ETH",
+  "rpc": ["https://rpc.testnet.mantle.xyz"],
+  "faucets": ["https://faucet.testnet.mantle.xyz"],
+  "nativeCurrency": {
+    "name": "Testnet BitDAO",
+    "symbol": "BIT",
+    "decimals": 18
+  },
+  "infoURL": "https://mantle.xyz",
+  "shortName": "mantle-testnet",
+  "chainId": 5001,
+  "networkId": 5001,
+  "explorers": [
+    {
+      "name": "Mantle Testnet Explorer",
+      "url": "https://explorer.testnet.mantle.xyz",
```

```suggestion
      "url": "https://explorer.mantle.xyz",
```

Seems that the testnet explorer is same as mainnet, can you update it?